src/geo/map/torque-layer.js

```diff
@@ -44,10 +49,18 @@ var TorqueLayer = LayerModelBase.extend({
   },
 
   _torqueTimeAttributeCartoCSSPropChanged: function () {
-    var regexp = /-torque-time-attribute:\s*["'](\w*)["']/;
-    var previousTorqueTimeAttributeProp = this.previous('cartocss').match(regexp) && this.previous('cartocss').match(regexp)[1];
-    var newTorqueTimeAttributeProp = this.get('cartocss').match(regexp) && this.get('cartocss').match(regexp)[1];
-    return newTorqueTimeAttributeProp !== previousTorqueTimeAttributeProp;
+    var currentCartoCSS = this.get('cartocss');
+    var previousCartoCSS = this.previous('cartocss');
+    var currentValue = this._getPropertyValueFromCartoCSS(currentCartoCSS, this.TORQUE_TIME_ATTRIBUTE_PROP);
```

Comment:
-torque-animation-duration is managed client side, -torque-resolution needs a map reload (it's the pixel resolution, it changes the aggregation)